### PR TITLE
Update maintenance mode message

### PR DESCRIPTION
**Pull request by trivikr**
Updates maintenance mode message to indicate it has entered maintenance mode

##### Checklist

- [x] changelog is added, `npm run add-change`

### CODE REVIEW

**Review by siddsriv — APPROVED** [review]

File: lib/maintenance_mode_message.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 var warning = [
-  'The AWS SDK for JavaScript (v2) will enter maintenance mode',
-  'on September 8, 2024 and reach end-of-support on September 8, 2025.\n',
+  'The AWS SDK for JavaScript (v2) is in maintenance mode.\n',
```
Comment (bkaws) [inline lib/maintenance_mode_message.js:2]: "... in maintenance mode and will no longer release features for this service."

Comment (bkaws) [reply lib/maintenance_mode_message.js:2]: The AWS SDK for JavaScript (v2) is in maintenance mode...
SDK releases are limited to address critical bug fixes and security issues only.

Comment (trivikr) [reply lib/maintenance_mode_message.js:2]:
```suggestion
  'The AWS SDK for JavaScript (v2) is in maintenance mode.',
  ' SDK releases are limited to address critical bug fixes and security issues only.\n',
```

**Review by kuhe — APPROVED** [review]

File: lib/maintenance_mode_message.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 var warning = [
-  'The AWS SDK for JavaScript (v2) will enter maintenance mode',
-  'on September 8, 2024 and reach end-of-support on September 8, 2025.\n',
+  'The AWS SDK for JavaScript (v2) is in maintenance mode.',
+  ' SDK releases are limited to address critical bug fixes and security issues only.\n',
   'Please migrate your code to use AWS SDK for JavaScript (v3).',
   'For more information, check blog post at https://a.co/cUPnyil'
```
Comment (bkaws) [inline lib/maintenance_mode_message.js:5]: check the blog post

Comment (trivikr) [inline lib/maintenance_mode_message.js:5]:
```suggestion
  'For more information, check the blog post at https://a.co/cUPnyil'
```

**Pull request merged**